addons/a11y/README.md

```diff
@@ -2,6 +2,8 @@
 
 This storybook addon can be helpfull to make your UI components more accessibile.
 
+[Framework Support](../../ADDONS_SUPPORT.md)
```

Comment:
Please make those links absolute, so that they would work on npm pages

Reply:
Like this: https://github.com/storybooks/storybook/blob/master/ADDONS_SUPPORT.md